### DISCUSSION TITLE
Improve keyword text search

### DISCUSSION THREAD

**Discussion by aviggiano** [solodit/feedback#9] [New Feature Requests]
The keyword text search does not always return existing values from the database.

I am not sure how it is implemented, but my impression is that it is a very basic string match. It would be better to improve it with fuzz search or similar methods.

For example, when I search for "liquidation frontrun", I get empty results:

![screenshot-2023-06-08-15-00-29](https://github.com/solodit/feedback/assets/3029017/644fe3a4-ad0d-4f56-b6f3-cd419ed532f4)

If I search for "liquidation", I get 269 results, and one of them contains a finding describing exactly a liquidation frontrunning issue:

![screenshot-2023-06-08-15-00-20](https://github.com/solodit/feedback/assets/3029017/86633a38-ac85-43a6-a5d6-c948b7524f40)

**Comment by hans-cyfrin** [comment]
Exact match is available now for your information.

**Reply by aviggiano** [reply]
Hello

Thanks for the response, although the initial suggestion was to implement a partial search. In any case I'm not sure if this is still relevant, this issues can probably be closed.

**Reply by hans-cyfrin** [reply]
We admit the search can be improved further. Hence will leave this open.
